README.md

```diff
@@ -1,3 +1,7 @@
 # DCP Prototype
 
 The Data Coordination platform (DCP) for the Human Cell Atlas (HCA) stores, organizes, serves, and enables the exploration of cell atlases.
+
+# Environment Variables
+
+`STAGE` - set this value to target a specific deployment stage for deploying code.
```

Comment:
A bunch of the other infra is already using 'DEPLOYMENT_STAGE'. If you are going to switch to stage, please change all references or use 'DEPLOYMENT_STAGE' for the browser as well

Reply:
will change